drivers/mpu9250/MPU9250.hpp

```diff
@@ -151,7 +151,8 @@ namespace DriverFramework
 #define BITS_ACCEL_CONFIG_16G		0x18
 
 // This is ACCEL_FCHOICE_B which is the inverse of ACCEL_FCHOICE
-#define BITS_ACCEL_CONFIG2_BW_1130HZ	0x08
+#define BITS_ACCEL_CONFIG2_BW_1046HZ	0x08
```

Comment:
You are checking an old RM, https://www.invensense.com/wp-content/uploads/2015/02/RM-MPU-9250A-00-v1.6.pdf , page 15